components/services/bat_ledger/public/cpp/manifest.cc

```diff
@@ -23,7 +23,9 @@ const service_manager::Manifest& GetManifest() {
                           kSharedAcrossGroups)
                   .WithExecutionMode(service_manager::Manifest::ExecutionMode::
                                          kOutOfProcessBuiltin)
+#if !defined(OS_ANDROID)
```

Comment:
why this change?

Reply:
it is way how it works on Android

Reply:
SandboxTyoe for ads is `utility`, I have not had a chance to check why ledger needs to be `none`. @NejcZdovc maybe you know?

Reply:
@bridiver good from your side? I don't have insight into it

Reply:
@SergeyZhukovsky Just checking but if Android crashes without the flag, is the #ifdef correct as it removes the flag for Android?

Reply:
yep, setting it to `none` works, but crashes without the whole ifdef section.